actionview/lib/action_view/helpers/form_options_helper.rb

```diff
@@ -112,7 +111,7 @@ module FormOptionsHelper
       # would become:
       #
       #   <select name="post[person_id]" id="post_person_id">
-      #     <option value=""></option>
+      #     <option value="" label=" "></option>
```

Comment:
This example is duplicated in the Action View Helpers guide, could you update it there too?

https://github.com/rails/rails/blob/b368511db24fb126545327984511281153730ec4/guides/source/action_view_helpers.md#L795